config/crd/bases/metal3.io_provisionings.yaml

```diff
@@ -43,6 +43,10 @@ spec:
               type: string
             provisioningNetwork:
               description: ProvisioningNetwork provides a way to indicate the state of the underlying network configuration for the provisioning network. This field can have one of the following values - `Managed`- when the provisioning network is completely managed by the Baremetal IPI solution. `Unmanaged`- when the provsioning network is present and used but the user is responsible for managing DHCP. Virtual media provisioning is recommended but PXE is still available if required. `Disabled`- when the provisioning network is fully disabled. User can bring up the baremetal cluster using virtual media or assisted installation. If using metal3 for power management, BMCs must be accessible from the machine networks. User should provide two IPs on the external network that would be used for provisioning services.
+              enum:
+              - Managed
+              - Unmanaged
+              - Disabled
               type: string
```

Comment:
Making the change in https://github.com/openshift/cluster-baremetal-operator/pull/19#discussion_r493020627 made these changes in 46 - 49 disappear.